### PR TITLE
[cosmetic] improve documentation and use proper verb for formatting

### PR DESCRIPTION
**Pull request by caglar10ur**

### CODE REVIEW

**Comment by cgtexmex** [comment]
lgtm

File: pkg/dio/multi.go [hunk 1]
```diff
@@ -39,20 +39,26 @@ type multiWriter struct {
...
+	var n int
+	var err error
+
 	// stash a local copy of the slice as we never want to write twice to a single writer
 	// if remove is called during this flow
 	wTmp := t.writers
 
 	if verbose {
-		log.Debugf("[%p] writing \"%s\" to %d writers", t, string(p), len(t.writers))
+		defer func() {
```
Comment (sflxn) [inline pkg/dio/multi.go]: defer a log message in a deferred function in an if-then block.  What's the desired behavior?

File: pkg/dio/multi.go [hunk 2]
```diff
@@ -147,7 +155,10 @@ type multiReader struct {
 func (t *multiReader) Read(p []byte) (int, error) {
 	n := 0
 	if verbose {
-		defer log.Debugf("[%p] read \"%s\" from %d readers", t, string(p[:n]), len(t.readers))
+		defer func() {
```
Comment (sflxn) [inline pkg/dio/multi.go]: same as above

File: pkg/serial/rawconn.go [hunk 3]
```diff
@@ -83,19 +83,25 @@ func NewRawConn(fd uintptr, name string, net string) (*RawConn, error) {
...
+// Read reads data from the connection.
+func (conn *RawConn) Read(b []byte) (int, error) {
 	defer trace.End(trace.Begin(""))
 
+	var n int
+	var err error
+
 	if verbose {
-		defer log.Debugf("Returning error and bytes from read (%s:%s): %d, %s", conn.rchannel.Name(), conn.wchannel.Name(), n, err)
+		defer func() {
```
Comment (sflxn) [inline pkg/serial/rawconn.go]: Same as above but no defer log.

File: pkg/dio/multi.go (hunk 1 above)
Comment (caglar10ur) [inline pkg/dio/multi.go]: Yep, we want the end result only if we have verbose set

Comment (caglar10ur) [inline pkg/dio/multi.go]: Oh wait, no, 2nd defer is fat fingerin (fixing now)

Comment (caglar10ur) [inline pkg/dio/multi.go]: Fixed...

File: pkg/dio/multi.go (hunk 2 above)
Comment (caglar10ur) [inline pkg/dio/multi.go]: Fixed...

File: pkg/serial/rawconn.go (hunk 3 above)
Comment (caglar10ur) [inline pkg/serial/rawconn.go]: Fixed...

**Comment by jzt** [comment]
👍  LGTM 👍

**Comment by sflxn** [comment]
lgtm

**Pull request merged**